components/Utils/src/formatDate.tsx

```diff
@@ -0,0 +1,50 @@
+import { format, differenceInDays } from 'date-fns';
+import { nl } from 'date-fns/locale';
+
+export interface FormatDateLabels {
+  today?: string;
+  yesterday?: string;
+  before?: string;
+  approachingDeadline?: (daysDifference: number) => string;
+}
+
+interface Props {
+  dateTime: string;
+  now?: string;
+  locale?: Locale;
+  labels?: FormatDateLabels;
+  relative?: boolean;
+}
+
+export const formatDate = ({
+  dateTime,
+  now = new Date().toISOString(),
+  locale = nl,
+  labels = {},
```

Comment:
Deze is optioneel. Wat gebeurt er als je die nu niet invult. Is niet handig om hier een default in te zetten?

Reply:
Ja goede, die moet niet optioneel zijn. Ik pas dat wel even aan.

Reply:
Done!